soc/arm/ene_kb1200/Kconfig.defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023 ENE Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_ENE_KB1200
+
+config SOC
+	default "ene_kb1200"
+
+config NUM_IRQS
+	default 64
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 48000000
```

Comment:
please, take this property from DT see eg https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/gd_gd32/gd32f4xx/Kconfig.defconfig.gd32f405#L10